src/vendor/jx/Edge.js

```diff
@@ -0,0 +1,36 @@
+/* eslint-disable linebreak-style */
+/* eslint-disable no-underscore-dangle */
```

Comment:
(here and in other files) I think that every time you use an `eslint-disable` directive, some documentation is needed saying why.  If this code is ever to be maintained by anyone other than yourself, it should adhere to the Principle of Least Surprise.  For example, what is it that `eslint` doesn't like about your linebreak-style, and why is it important?

Reply:
Yes, that is an embarrassment you get to see with this WIP: I work in windows, and my IDE refuses to admit there are `\r` in my source files, even though there are. Git eventually scrubs them out, and I can remove this.

Even if this was permanent disabling: I would assume the file might not conform to `linebreak-style`, and I would assume it is unimportant because all linting is unimportant: Someone, somewhere wanted to disable linting rather than add more code to satisfy a linter.

I am still new at working for an aggressive linter.  So far, my experience is it provides copious amounts of low quality feedback, which pollutes the console output during debugging and interfering with finding legitimate errors. For me, each `eslint-disable` line is a signal that someone escaped oppression. :)

Seriously, I continue to look for a solution.